frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -197,12 +242,35 @@ export function PrintTestDeckScreen(): JSX.Element {
       await makeCancelable(sleep(numPages * ONE_SIDED_PAGE_PRINT_TIME_MS));
       setPrintIndex({
         precinctIndex: printIndex.precinctIndex,
-        component: 'HandMarkedPaperBallots',
+        component: 'BmdPaperBallots',
       });
     },
     [printIndex, printer, logger, currentUserType, precinctIds, makeCancelable]
   );
 
+  const onAllBmdPaperBallotsRendered = useCallback(
+    async (numBallots) => {
+      if (!printIndex) {
+        return;
+      }
+
+      const precinctId = precinctIds[printIndex.precinctIndex];
+      await printer.print({ sides: 'one-sided' });
+      await logger.log(LogEventId.TestDeckPrinted, currentUserType, {
+        disposition: 'success',
+        message: `BMD paper ballot test deck printed as part of L&A package for precinct ID: ${precinctId}`,
+        precinctId,
+      });
+
+      await makeCancelable(sleep(numBallots * ONE_SIDED_PAGE_PRINT_TIME_MS));
```

Comment:
We're planning to test with real hardware to determine whether our print time estimates are accurate. This will become especially important as we introduce the ability to print to different trays depending on paper size

https://github.com/votingworks/vxsuite/issues/1865
https://github.com/votingworks/vxsuite/issues/1861